EscelForce.Business.Login/Services/MapExtraction/UpdateExtractionMapService.cs

```diff
@@ -469,16 +481,18 @@ public ServiceResponseModel<FieldSelectionModel> SubmitForNewChild(SearchSortExt
 
                 //var currentObject = _persistenceContainer.Get<string>(BusinessConstants.CurrentObject);
 
-                var sfObject = queryObject?.Objects.FirstOrDefault(x => x.Name == model.SelectedChild);
+               // var sfObject = queryObject?.Objects.FirstOrDefault(x => x.Name == model.SelectedChild);
```

Comment:
Remove all commented code if not needed anymore.

Reply:
@ritwiksen removed